ganga/GangaCore/Runtime/GPIFunctions.py

```diff
@@ -93,3 +95,9 @@ def convert_merger_to_postprocessor(j):
         mp.process_objects.append(stripProxy(j).merger)
         stripProxy(j).postprocessors = mp
 
+def runfile(path_to_file):
+    """ A wrapper for the runpy.run_path() function. """
+    if not type(path_to_file) == str:
```

Comment:
Avoid direct type comparisons. Always use `if not isinstance(path_to_file, str):` instead as somebody might in principle have a class that inherits from `str`.